bigchaindb/tendermint/lib.py

```diff
@@ -191,6 +191,22 @@ def get_block(self, block_id, include_status=False):
         else:
             return block
 
+    def get_blocks_status_containing_tx(self, txid):
```

Comment:
Since we have instant finality, I'd keep the `status` out of the tendermint module. It's OK to change the code in ` bigchaindb/web/views/blocks.py` to call another method.